R/retry.R

```diff
@@ -93,21 +93,23 @@ retry_should_terminate <- function(i, times, resp, terminate_on, terminate_on_su
 backoff_full_jitter <- function(i, resp, pause_base = 1, pause_cap = 60,
                                 pause_min = 1, quiet = FALSE) {
   length <- max(pause_min, stats::runif(1, max = min(pause_cap, pause_base * (2^i))))
-  if (!quiet) {
-    if (inherits(resp, "error")) {
-      error_description <- gsub("[\n\r]*$", "\n", as.character(resp))
-      status <- "ERROR"
-    } else {
-      error_description <- ""
-      status <- status_code(resp)
-    }
-    if (status == 429) {
-      retry_after <- resp$headers[["retry-after"]]
-      if (!is.null(retry_after)) {
-        length <- max(pause_min, as.numeric(retry_after))
-      }
+  if (inherits(resp, "error")) {
+    error_description <- gsub("[\n\r]*$", "\n", as.character(resp))
+    status <- "ERROR"
+  } else {
+    error_description <- ""
+    status <- status_code(resp)
+  }
+  if (status == 429) {
+    retry_after <- resp$headers[["retry-after"]] %||%
+      resp$headers[["Retry-After"]]
```

Comment:
I'm all for the change to make this case-insensitive. But I think we might as well go all the way and use an approach that would match against, e.g., `Retry-after`, as well. Not that I expect that specifically, more on principle.

Reply:
Reading more, perhaps this is encompassed by @hadley's comment, in fact.

Reply:
Maybe what you need to do @patr1ckm is search for other uses of `insensitive()` here and incorporate that into your PR.